### PR TITLE
[frontend] sort exports by export date in lists (#5316)

### LINKED ISSUE #5316: Export list displays exported files in disorder
## Description

When you export a list of entities, you should see the list of exported files in order of export date (newest first), but they are displayed in random order. Which can be confusing when you do an export, you will see the export in progress on top, then it disapears in the list, as if it didn't work, but it is just in the middle of the list.

image

## Environment

1. OS (where OpenCTI server runs): { e.g. Mac OS 10, Windows 10, Ubuntu 16.4, etc. }
2. OpenCTI version: 5.12.14
3. OpenCTI client: frontend
4. Other environment details:

### PR DESCRIPTION
**Pull request by lndrtrbn**
### Proposed changes

* Sort exported files by lastModified attribute

### Related issues

* #5316 

### Checklist

- [x] I consider the submitted work as finished
- [x] I tested the code for its functionality
- [ ] I wrote test cases for the relevant uses case
- [ ] I added/update the relevant documentation (either on github or on notion)
- [ ] Where necessary I refactored code to improve the overall quality

### Further comments

If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

File: opencti-platform/opencti-front/src/private/components/common/stix_core_objects/StixCoreObjectsExportsContent.jsx [hunk 1]
```diff
@@ -33,7 +33,8 @@ const StixCoreObjectsExportsContentComponent = ({
       subscription.unsubscribe();
     };
   });
-  const stixCoreObjectsExportFiles = data?.stixCoreObjectsExportFiles?.edges ?? [];
+  const stixCoreObjectsExportFiles = [...(data?.stixCoreObjectsExportFiles?.edges ?? [])]
+    .sort((f1, f2) => f2?.node?.lastModified?.localeCompare(f1?.node?.lastModified));
```
Comment (SouadHadjiat) [inline opencti-platform/opencti-front/src/private/components/common/stix_core_objects/StixCoreObjectsExportsContent.jsx]: if we sort only in the frontend, we might not get the whole list of exported files (and not get the latest ones). the API should return the ordered list. 
What is strange in the current implementation is that we only get 25 files, without pagination. So if our last export is not in them, we won't see it.

Comment (lndrtrbn) [reply opencti-platform/opencti-front/src/private/components/common/stix_core_objects/StixCoreObjectsExportsContent.jsx]: Oh yes.. I'll have a look in the backend instead to change this

**Comment by richard-julien** [comment]
Force pushed for a default backend approach. Please take a look @lndrtrbn

**Comment by lndrtrbn** [comment]
Tested, seems good to me. Better solution than I suggested 😂
Can you have a look @SouadHadjiat ?

**Review by Kedae — APPROVED** [review]

**Pull request merged**